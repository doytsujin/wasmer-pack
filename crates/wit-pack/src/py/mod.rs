use anyhow::Error;

use crate::{Files, Interface, Metadata, Module};

/// Generate Python bindings.
pub fn generate_python(
    _metadata: &Metadata,
    _module: &Module,
    _interface: &Interface,
) -> Result<Files, Error> {
    todo!()
}
