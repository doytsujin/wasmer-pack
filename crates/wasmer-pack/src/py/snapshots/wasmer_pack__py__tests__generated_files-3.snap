---
source: crates/wit-pack/src/py/mod.rs
expression: "files[\"wasmer_pack/__init__.py\"].utf8_contents().unwrap().replace(crate::GENERATOR,\n    \"XXX\")"
---
'''
Bindings to wasmer/wasmer-pack v1.2.3.
'''

# Generated by XXX.

from wasmer import Store as _Store # type: ignore
from .commands import Commands as _Commands
from .bindings import Bindings as _Bindings

__version__ = "1.2.3"

_store = _Store()
bindings = _Bindings(_store)
commands = _Commands(_store)
